AutoDI.Build/AssemblyRewriteTask.cs

```diff
@@ -209,6 +218,15 @@ string FindPdbPath()
                 return null;
             }
 
+            bool IsPortablePdb(string symbolsPath)
+            {
+                using (var fileStream = File.OpenRead(symbolsPath))
+                using(var reader = new BinaryReader(fileStream))
+                {
+                    return reader.ReadBytes(4).SequenceEqual(new byte[] {0x42, 0x4a, 0x53, 0x42});
```

Comment:
Is this byte sequence specified somewhere, or is this something you found by pulling apart PDBs?